src/config/yaml.ts

```diff
@@ -29,6 +29,10 @@ try {
 export const yamlConfig = merge(defaultConfig, customConfig)
 
 const ajv = new Ajv()
+
+// TODO: fix errors
+// const ajv = new Ajv({ allErrors: true, strict: "log" })
+
```

Comment:
Please clean

Reply:
the comment is voluntary. the current schema.ts validation has issue that are been shown with `strict: "log"`, but me or someelse should cleaned up later